### ISSUE TITLE
feature request: mobile web browser no-scroller, auto fit feature to desktop version please

### ISSUE THREAD

**Issue by bryanray** [AlUlkesh/stable-diffusion-webui-images-browser#133]
feature request: mobile web browser no-scroller, auto fit feature to desktop version please

**Comment by AlUlkesh** [comment]
:dizzy_face: ... what? ELI5 please.

**Comment by bryanray** [comment]
![IMG_3604](https://user-images.githubusercontent.com/1768531/230899419-b4158513-4e10-40fa-bf56-7d7fddeb3e9e.jpg)

you know, in the mobile web browser version, it has no scroller, just shows all images,
BUT 
in desktop page, it will never show all images, it will always let you scroll inside that "box".

**Comment by AlUlkesh** [comment]
I didn't know that was the case in mobile. Interesting.

For desktop we've been trying to enable it for some time, but no luck so far:
#40 
#47

**Comment by robonxt** [comment]
Interesting. On my mobile and desktop devices (Android 12 phone +tablet, Windows 11 laptop + desktop), there is a scrollbar, but it looks custom (not like @bryanray 's scrollbar)
